wiki/osu!_File_Formats/Db_(file_format)/en.md

```diff
@@ -179,6 +179,4 @@ This database contains the scores achieved locally.
 | Int | Should always be 0xffffffff (-1). |
 | Long | Online Score ID |
 
-Apart from the online score ID, the individual score format is the same as the replay format. [Osr (file format)][Osr Link]. This explains the empty string and -1 int.
-[Osr Link]: /wiki/osu!_File_Formats/Osr_(file_format)
-[Osu Link]: /wiki/osu!_File_Formats/Osu_(file_format)
+Apart from the online score ID, the individual score format is the same as the replay format. This explains the empty string and -1 int. See [this](/wiki/osu!_File_Formats/Osr_(file_format)).
```

Comment:
`See [this](/wiki/osu!_File_Formats/Osr_(file_format))` -> `For more information, see [.osr (file format)](/wiki/osu!_File_Formats/Osr_(file_format)).`